README.md

```diff
@@ -10,7 +10,18 @@ with **simple steps** people can **_easily_ follow** <br />
 **keep our _communication consistently crystal clear_**! <br />
 
 Our contribution steps in a _nutshell_:
-
+<!-- The text under the magnifying glass says: 
+        "validate the need exists"
+        "or undesired behavior"
+        "from the user's perspective"
+
+      I believe this should be something like
+        "validate the need or"
```

Comment:
Agreed. the text under the magnifying glass is not _complete_.
![validate-idea](https://cloud.githubusercontent.com/assets/194400/22287688/a929a9ae-e2eb-11e6-8d16-f2ebbebea080.png "DWYL's Contribution Guidelines Steps Diagram")
The image was created in Paint. 😜 
If you have _any_ image editor, grab the original and modify it. 
(_not urgent for **`this`** PR, but if you're feeling that way inclined in future..._) 👍